src/App.tsx

```diff
@@ -58,7 +58,9 @@ const App = () => {
 
 	const backgroundColor = urlParams.get('bg') || process.env.DESCOPE_BG_COLOR;
 
-	const faviconUrl = process.env.REACT_APP_FAVICON_URL || '';
+	const faviconUrl =
```

Comment:
Just notice that existing customers might get console error if there is no favicon.
Can you check that?

Reply:
no console errors - just checked @dorsha

Reply:
What about network error? 
Can you paste here a screenshot of the network when the URL does not exist?

Reply:
<img width="646" alt="image" src="https://github.com/user-attachments/assets/60f7d76f-b404-4d5e-a9a1-c5553fd71562">
<img width="633" alt="image" src="https://github.com/user-attachments/assets/ead48616-bc6c-4bf0-97ea-d8e64f68a572">
<img width="644" alt="image" src="https://github.com/user-attachments/assets/0bab17c7-5b2e-4c4f-ae01-9f291404a9ad">

Reply:
So this is a regression.

Reply:
Can we know if an image is there? Maybe we should have default image for all projects/apps.

Reply:
@dorsha why would it be a regression?

Reply:
we already have a default favicon in the app that is used 🤔 
i don't think there's a need to duplicate data and upload the same default favicon every time

Reply:
Today, customers don't have any network error.
And now after your change they will get a network error, right?